pkg/storaged/utils.js

```diff
@@ -767,7 +767,31 @@ export function find_children_for_mount_point(client, mount_point, self) {
     return children;
 }
 
-export function get_active_usage(client, path, top_action, child_action) {
+export function get_fstab_config_with_client(client, block, also_child_config) {
+    let config = block.Configuration.find(c => c[0] == "fstab");
+
+    if (!config && also_child_config && client.blocks_crypto[block.path])
+        config = client.blocks_crypto[block.path]?.ChildConfiguration.find(c => c[0] == "fstab");
+
+    if (config && decode_filename(config[1].type.v) != "swap") {
+        const mnt_opts = get_block_mntopts(config[1]).split(",");
+        let dir = decode_filename(config[1].dir.v);
+        let opts = mnt_opts
+                .filter(function (s) { return s.indexOf("x-parent") !== 0 })
+                .join(",");
+        const parents = mnt_opts
+                .filter(function (s) { return s.indexOf("x-parent") === 0 })
+                .join(",");
+        if (dir[0] != "/")
+            dir = "/" + dir;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19418-20231023-134501-b6ea4635-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)